gdm/PKGBUILD

```diff
@@ -0,0 +1,184 @@
+        'f7e868fdd7cc121433de1572583eb728f4d186cd4f52c6d6c8f2ccf4a3cf781144ff71f704f13571ddb97a1ff4ec55cfa3df25d38737ad19da21e84ddc2d3ee4')
+
+prepare() {
+  cd gdm
+
+  # data: Re-enable wayland on hybrid graphics laptops with NVIDIA driver
+  git cherry-pick -n 4291dde66240ac23ee3216fe10fdd6f9d302e21c
+
+  # data: Revert part of wayland hybrid graphics change
+  git cherry-pick -n 8211f6d39578e0e8756ded098433ad79fbc76670
+
+  # Don't start ssh-agent by default
+  git apply -3 ../0001-Xsession-Don-t-start-ssh-agent-by-default.patch
+}
+
+build() {
+  local meson_options=(
+    -D dbus-sys="/usr/share/dbus-1/system.d"
+    -D default-pam-config=arch
+    -D default-path="/usr/local/bin:/usr/local/sbin:/usr/bin"
+    -D gdm-xsession=true
+    -D ipv6=true
+    -D run-dir=/run/gdm
+    -D selinux=disabled
+  )
+
+  arch-meson gdm build "${meson_options[@]}"
+  meson compile -C build
+}
+
+check() {
+  meson test -C build --print-errorlogs
+}
+
+_pick() {
+  local p="$1" f d; shift
+  for f; do
+    d="$srcdir/$p/${f#$pkgdir/}"
+    mkdir -p "$(dirname "$d")"
+    mv "$f" "$d"
+    rmdir -p --ignore-fail-on-non-empty "$(dirname "$f")"
+  done
+}
+
+package_gdm() {
+  depends+=(libgdm)
+  optdepends=('fprintd: fingerprint authentication')
+  backup=(
+    etc/gdm/PostSession/Default
+    etc/gdm/PreSession/Default
+    etc/gdm/Xsession
+    etc/gdm/custom.conf
+    etc/pam.d/gdm-autologin
+    etc/pam.d/gdm-fingerprint
+    etc/pam.d/gdm-launch-environment
+    etc/pam.d/gdm-password
+    etc/pam.d/gdm-smartcard
+  )
+  groups=(gnome)
+  install=gdm.install
+
+  meson install -C build --destdir "$pkgdir"
+
+  cd "$pkgdir"
+
+  install -d -o   0 -g   0 -m 0755 var
+  install -d -o   0 -g   0 -m 0755 var/lib
+  install -d -o 120 -g 120 -m 1770 var/lib/gdm
+  install -d -o 120 -g 120 -m 0700 var/lib/gdm/.config
+  install -d -o 120 -g 120 -m 0700 var/lib/gdm/.config/pulse
+  install -d -o 120 -g 120 -m 0700 var/lib/gdm/.local
+  install -d -o 120 -g 120 -m 0755 var/lib/gdm/.local/share
+  install -d -o 120 -g 120 -m 0755 var/lib/gdm/.local/share/applications
+
+  # https://src.fedoraproject.org/rpms/gdm/blob/master/f/default.pa-for-gdm
+  install -o120 -g120 -m644 /dev/stdin var/lib/gdm/.config/pulse/default.pa <<END
+load-module module-device-restore
+load-module module-card-restore
+load-module module-udev-detect
+load-module module-native-protocol-unix
+load-module module-default-device-restore
+load-module module-always-sink
+load-module module-intended-roles
+load-module module-suspend-on-idle
+load-module module-systemd-login
+load-module module-position-event-sounds
+END
+
+  install -Dm644 /dev/stdin usr/lib/sysusers.d/gdm.conf <<END
+g gdm 120 -
+u gdm 120 "Gnome Display Manager" /var/lib/gdm
+END
+
+  install -Dm644 /dev/stdin usr/share/glib-2.0/schemas/org.gnome.login-screen.gschema.override <<END
+[org.gnome.login-screen]
+logo='/usr/share/pixmaps/archlinux-logo-text-dark.svg'
```

Comment:
Maybe we can override here to CachyOS Logo.

Reply:
Yeah, sounds good

Reply:
Hmm, looks like we need add dependency to `cachyos-settings` for this, since `/usr/share/icons/cachyos.svg` is owned by it.

Reply:
I think we could put then an own package for that?
I think EOS made it like that, so we dont need to maintain all time an own gdm.

Reply:
> 

Well, we can also put ``/usr/share/glib-2.0/schemas/aa_cachyos.org.gnome.login-screen.gschema.override`` in cachyos-settings

Reply:
Done with https://github.com/CachyOS/CachyOS-Settings/pull/78